### DISCUSSION TITLE
⛔ 🛟 Support Request: Naive question: building Elementor

### DISCUSSION THREAD

**Discussion by gary-schiltz** [elementor/elementor#21586] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would post this question in a general category, but there only seems to be a feature request category.

I'm pretty experienced with working with open source as hosted on GitHub, but I have next to no experience working on WordPress plugins, or PHP for that matter. So the question is if there is some sort of build script to turning this tree of source code into a WordPress zip file that can be installed from the dashboard?

### Describe the solution you'd like

It would be nice if there was some discussion of how to geting started making changes to Elementor core.

**Comment by m1ga** [comment]
check the package.json file. There are the build, test,.. scripts.
and the .github folder https://github.com/elementor/elementor/tree/master/.github/scripts if you want some other scripts. But the node part is fine, build a test version of 3.12.0 two weeks ago to [test the topbar](https://medium.com/@miga_/early-look-at-the-new-editor-v2-in-elementor-3-12-0-master-branch-2821e15a5240)
